oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -733,6 +803,13 @@ public Builder setWorkforcePoolUserProject(String workforcePoolUserProject) {
       return this;
     }
 
+    /** Sets the optional service account impersonation options. */
+    public Builder setServiceAccountImpersonationOptions(
```

Comment:
ServiceAccountImpersonationOptions is package private. Should either be public or pass a map here?